pkg/services/ngalert/notifier/available_channels.go

```diff
@@ -863,6 +863,21 @@ func GetAvailableNotifiers() []*alerting.NotifierPlugin {
 					PropertyName: "apiUrl",
 					Required:     true,
 				},
+				{
+					Label:          "Message",
+					Description:    "Alert text limited to 130 characters.",
+					Element:        alerting.ElementTypeInput,
+					InputType:      alerting.InputTypeText,
+					Placeholder:    `{{ template "default.title" . }}`,
+					PropertyName:   "message",
+					ValidationRule: "^.{0,130}$",
```

Comment:
This validation rule will not be effective as it can give false positives and negatives with templated values. The template string can technically be longer than 130 characters while the templated value is shorter than 130 (and vice versa).

I would recommend to remove it and add a note (maybe in the description) that it will be truncated if above 130 characters. This is similar to how we handle pagerduty though it doesn't explicitly mention it in the description.

Reply:
Oh you're right! Good catch, I'll fix this.